02/components/App.js

```diff
@@ -2,16 +2,8 @@
 import React from 'react';
 import Box from './Box';
 
-class App extends React.Component {
-    state = {
-        text: 'React HelloWorld Modern!',
-    };
-
-    render() {
-        const { text } = this.state;
-
-        return <Box text={text} />;
-    }
+function App () {
+    return <Box />;
```

Comment:
Miałeś przekazać wartość `state` przez kontekst więc tutaj powinien pojawić się `Provider`.
Wartość domyślna to co innego, którą zdefiniowałeś w `context.js` ;)